### ISSUE TITLE
Sign up for a volunteer account

### ISSUE THREAD

**Issue by azeng8100** [GTBitsOfGood/VolunTrack#14]
### Description
* Previously there was an investigation ticket: #6 
  * tl;dr No new user can be created since they always receive an "Unauthorized User" error
* The goal of this ticket is to fix the existing authorization code
* **NOTE: If the code is unreadable or you cannot find the bug, remake the entire authorization code flow with auth0 or something similar**
* Volunteers must have an existing account in order to register for an event. If they log in with a Google account that is not associated with an existing account, prompt them to provide their information
  * The prompt page is not as important as figuring out the authorization issue
  * The prompt page can look like basic HTML as long as it is functionally working. Designers will provide a design at a later time and it can be fixed then
### Acceptance Criteria
* New volunteers are prompted and fields are parsed correctly
* MongoDB looks correct

**Comment by azeng8100** [comment]
Moved to sprint 3. Serena and Phoebe assigned a different task. Jack will finish up what's left of this ticket
